### DISCUSSION TITLE
domino/12.0.2/admin/tune_runningcompactusingaprogramdocument_t

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#88] [Q&A]
# domino/12.0.2/admin/tune_runningcompactusingaprogramdocument_t

Use a Program document to schedule Compact to run at a regular time. For example, schedule Compact to run with the -B option once a week.

https://help.hcl-software.com/domino/12.0.2/admin/tune_runningcompactusingaprogramdocument_t.html

**Comment by Davehabz** [comment]
This would be more effective if it also included screenshots of examples of common program documents used, such as compacting Admin4.nsf.

**Reply by gthiel-id** [reply]
Thanks for your feedback. Putting it in the queue for doc improvements.
